hangups/message_parser.py

```diff
@@ -8,16 +8,19 @@
 
 
 # Common regex patterns
-boundary_chars = r'\s`!()\[\]{{}};:\'".,<>?«»“”‘’*_~='
-b_left = r'(?:(?<=[' + boundary_chars + r'])|(?<=^))'  # Lookbehind
-b_right = r'(?:(?=[' + boundary_chars + r'])|(?=$))'  # Lookahead
+BOUNDARY_CHARS = r'\s`!()\[\]{{}};:\'".,<>?«»“”‘’*_~='
+B_LEFT = r'(?:(?<=[' + BOUNDARY_CHARS + r'])|(?<=^))'  # Lookbehind
+B_RIGHT = r'(?:(?=[' + BOUNDARY_CHARS + r'])|(?=$))'   # Lookahead
+del BOUNDARY_CHARS
```

Comment:
Can you remove the `del` statements? I don't think they're worthwhile.

Reply:
Done in https://github.com/tdryer/hangups/pull/381/commits/5d4edcacd865257b1361efc1e5dc52511ca04013.